### ISSUE TITLE
HWE_P in run_eggla_gwas() results inconsistent with analysis plan 

### ISSUE THREAD

**Issue by annihei** [mcanouil/eggla#91]
### Bug description

These are from the older eggla version as I haven't got updated results yet, but in case you haven't noticed this (I didn't find in change log) the result formatting in run_eggla_gwas leaves values for HWE_P in every SNP when the analysis plan stated to set HWE_P missing for imputed SNPs.  

I didn'pay attention to this when I checked the NFBC results and corrected it, but now found the same in OBE results:
```
> summary(OBE_AR$HWE_P) # there are no missing values
   Min. 1st Qu.  Median    Mean 3rd Qu.    Max.
 0.0000  1.0000  1.0000  0.8859  1.0000  1.0000

```

### `eggla` version output

```eggla_0.17.4```  and older?

### Checklist

- [X] [formatted your issue](https://yihui.org/issue/#please-format-your-issue-correctly) so it is easier for us to read?
- [X] included a minimal, fully reproducible example in a single .qmd file? Please provide the whole file rather than the snippet you believe is causing the issue.
- [X] documented the version of `eggla` you're running, by pasting the output from running `packageVersion('eggla')` in the "`eggla` version output" text area?
- [X] documented which operating system you're running? If on Linux, please provide the specific distribution as well.

**Comment by annihei** [comment]
Thinking about this, actually, I am not sure if the idea is that eggla package user won't have to edit the summary stats file at all. If the idea is that they check it and do final editing themselves, then maybe it could be clearly stated in the analysis plan. If the idea is that eggla would provide ready to send results, then I think there might be inconsistency also with file delimiter (comma vs tab), and the columns it provides (maybe we'll get to the columns suggested in the analysis plan when we get further with the meta-analysis).

**Comment by mcanouil** [comment]
Oh, I missed the part where HWE_P should be NA for imputed variants.
I can fix it.

The goal in my opinion is to make "ready to send" tables, but this does not mean the analysts should not check what they are going to send. They *HAVE* to check, because they are the only one to have the ability to check if the results is consistent with the data.
Indeed, maybe a "danger" callout fully stating that results check is mandatory can be added in the analytical plan.

**Comment by mcanouil** [comment]
@annihei I will fix it soon.

**Comment by annihei** [comment]
I agree results should always be checked before sending anyway.

**Comment by mcanouil** [comment]
@annihei Fix available in v0.18.3

**Comment by annihei** [comment]
Thank you!
